cloudbuild/nightly/cloudbuild.yaml

```diff
@@ -99,28 +99,10 @@ steps:
       - 'CLUSTER_TABLE_API_UNBOUNDED_TABLE_TEST_FILE=${_CLUSTER_TABLE_API_UNBOUNDED_TABLE_TEST_FILE}'
       - 'WORKER_MACHINE_TYPE_UNBOUNDED=${_WORKER_MACHINE_TYPE_UNBOUNDED}'
 
-# 4. Start the simple table e2e test.
```

Comment:
Any specific reason to remove the simple test. Shouldn't have atleast one simple test? In case nested check fails in future.

Reply:
The simple table test was good while starting out, but now its redundant and doesnt add any value. If nested check fails in the future then we have to debug anyway.